edx/frontend-app-discussions
# feat: Update the post editor to include TinyMCE and connect to API [BD-38] [TNL-7353]  [BB-2900]

This change adds the TinyMCE editor for editing posts and connects the post editor to the API, allowing adding new posts and editing existing ones.


 **Creating a new post:**

https://user-images.githubusercontent.com/118837/131846368-b827ac90-c9be-48a4-be85-b9f87a62993e.mp4

**Editing an existing post:**
![Screenshot 2021-09-02 at 12-47-50 Discussions localhost](https://user-images.githubusercontent.com/118837/131846319-a7238891-34fd-4f83-97aa-e8d7a64f5942.png)


**Test instructions:**
1. Try creating a new post using the UI
2. After creating the post, you should be redirected to the post after a second or so
3. You can edit and existing post by visiting it, and changing the URL to append /edit 

A future PR will add proper edit buttons etc.
